### DISCUSSION TITLE
How to add new FECs?

### DISCUSSION THREAD

**Discussion by TrevorYFW** [Mr-MIBonk/M.I.B._More-Incredible-Bash#49] [Q&A] [answered]
Hi, just trying to update ROW maps where Australia is supposed to be, but it kept saying that no valid updates found on the medium no matter what methods I have tried, so I think there might be an issue with the FECs patched. After looking through the internet, I think the valid FECs for ROW might be 023600ee and 023d00ee. I added those two in addfecs.txt and also the txt file under my version - corresponding folder under "patches", but it only patched all the other Fecs except the ones I manually added. Is there another file I need to add in those two FECs in order to get them patched?

Thanks

**Comment by harman-f** [comment, marked as answer]
You can add any FEC you want to have included into your FWs addfec.txt on SD card.

So like you explained yourself.

021D00EE is correct.

**Reply by TrevorYFW** [reply]
Yes I have added it in the txt file but it didnt show up after patching it. All the other FECs on the txt file showed up though..

**Reply by harman-f** [reply]
Just run addFec function again in M.I.B

**Reply by TrevorYFW** [reply]
> Just run addFec function again in M.I.B

Didn't know that I could use this function...all sorted now! Thanks

**Comment by Mr-MIBonk** [comment]
023D00ee is correct...

**Comment by Mr-MIBonk** [comment]
023D00ee is correct...
